org.adempiere.base/src/org/compiere/model/MOrder.java

```diff
@@ -1541,6 +1541,12 @@ public String prepareIt()
 			return DocAction.STATUS_Invalid;
 		}
 		
+		if (!MAcctSchema.isBackDateTrxAllowed(getCtx(), getDateAcct()))
+		{
+			m_processMsg = "@BackDateTrxNotAllowed@";
```

Comment:
this perhaps not needed since order has no accounting and costing consequences.

Reply:
Removed the validation